src/Controllers/SettingsController.php

```diff
@@ -96,6 +96,49 @@ public function savePassword(Request $request): Response
         return $this->redirect->to('/settings/password');
     }
 
+    /**
+     * @return Response
+     */
+    public function theme(): Response
+    {
+        $themes = array_map(function ($theme) {
+            return $theme['name'];
+        }, config('themes'));
+
+        $currentTheme = $this->auth->user()->settings->theme;
+
+        return $this->response->withView(
+            'pages/settings/theme',
+            [
+                'settings_menu' => $this->settingsMenu(),
+                'themes' => $themes,
+                'current_theme' => $currentTheme
+            ] + $this->getNotifications()
+        );
+    }
+
+    /**
+     * @param Request $request
+     * @return Response
+     */
+    public function saveTheme(Request $request): Response
+    {
+        $user = $this->auth->user();
+        $data = $this->validate($request, ['select_theme' => 'int']);
+        $selectTheme = $data['select_theme'];
+
+        if (!array_key_exists($selectTheme, config('themes'))) {
```

Comment:
a `if (isset(config('themes')[$selectTheme]))` would be also feasible but the explicit form is also good here